include/cuco/static_multiset_ref.cuh

```diff
@@ -200,6 +200,32 @@ class static_multiset_ref
   template <typename... NewOperators>
   [[nodiscard]] __host__ __device__ auto with(NewOperators... ops) && noexcept;
```

Comment:
Shall we also add a `with_operators` alias and mention in the docs of `with` that it's deprecated and will be removed in the future?